infra/base-images/base-builder/compile_go_fuzzer

```diff
@@ -48,7 +48,14 @@ if [[ $SANITIZER = *coverage* ]]; then
   abspath_repo=`go list -m $tags -f {{.Dir}} $fuzzed_repo || go list $tags -f {{.Dir}} $fuzzed_repo`
   # give equivalence to absolute paths in another file, as go test -cover uses golangish pkg.Dir
   echo "s=$fuzzed_repo"="$abspath_repo"= > $OUT/$fuzzer.gocovpath
-  go test -run Test${function}Corpus -v $tags -coverpkg $fuzzed_repo/... -c -o $OUT/$fuzzer $path
+  # additional packages for which to get coverage
```

Comment:
Please begin comments with capital and end with period.

Reply:
ok